lib/src/cmaf/ham/mapper/hls/mapHlsToHam.ts

```diff
@@ -43,15 +82,17 @@ function mapHlsToHam(manifest: Manifest) {
 		// TODO: retrieve channels, samplerate, bandwidth and codec
 		audioTracks.push({
 			id: audio,
-			type: 'AUDIO',
+			type: 'audio',
 			name: uri,
-			codec: '',
-			duration: targetDuration,
+			// Using codec mp4a.40.2 for now, we should retrieve it by finding
+			// the video playlist that is related to this audio group.
+			codec: 'mp4a.40.2',
+			duration: targetDuration * segments.length,
 			language: language,
 			bandwidth: 0,
 			segments: segments,
 			sampleRate: 0,
-			channels: 0,
+			channels: 2,
```

Comment:
Same comment here, this harcoded value should be explained

Reply:
I updated the TODO comment that you already put here, so we have just one comment.